doc/src/manual/constructors.md

```diff
@@ -481,26 +491,26 @@ Following the outer constructor definitions, we have a number of methods for the
 operator, which provides a syntax for writing rationals. Before these definitions, [`//`](@ref)
 is a completely undefined operator with only syntax and no meaning. Afterwards, it behaves just
 as described in [Rational Numbers](@ref) -- its entire behavior is defined in these few lines.
-The first and most basic definition just makes `a//b` construct a `Rational` by applying the
-`Rational` constructor to `a` and `b` when they are integers. When one of the operands of [`//`](@ref)
+The first and most basic definition just makes `a//b` construct a `OurRational` by applying the
+`OurRational` constructor to `a` and `b` when they are integers. When one of the operands of [`//`](@ref)
 is already a rational number, we construct a new rational for the resulting ratio slightly differently;
-this behavior is actually identical to division of a rational with an integer. Finally, applying
-[`//`](@ref) to complex integral values creates an instance of `Complex{Rational}` -- a complex
+this behavior is actually identical to division of a rational with an integer.
+Finally, applying
+[`//`](@ref) to complex integral values creates an instance of `Complex{OurRational}` -- a complex
 number whose real and imaginary parts are rationals:
 
-```julia
-julia> (1 + 2im)//(1 - 2im)
--3//5 + 4//5*im
```

Comment:
I mean, we could, but that seems like a lot of work to show one line, when the point is the subtyping fun.